docs/mp/openapi/01_openapi.adoc

```diff
@@ -26,6 +26,7 @@
 :mp-openapi-prefix: https://github.com/eclipse/microprofile-open-api/blob/master
 :mp-openapi-spec: {mp-openapi-prefix}/spec/src/main/asciidoc/microprofile-openapi-spec.adoc
```

Comment:
The links to the MP OpenAPI spec should change:

1. We need to use the actual version we depend on `3.0` not `master`. Ideally this would be set in an AsciiDoc attribute so it's easily changed.
2. MP changed the file type of the spec source from `.adoc` to `.asciidoc`. 

I think those are the only link corrections needed but after revising them please double-check that clicking on them works.